armotypes/vulnerabilitytypes.go

```diff
@@ -9,11 +20,100 @@ type VulnerabilityJobParams struct {
 	JobID           string `json:"jobID,omitempty"`
 }
 
+type VulnerabilityWorkload struct {
+	Wlid             string                     `json:"wlid"`
+	Name             string                     `json:"name"`
+	Namespace        string                     `json:"namespace"`
+	Kind             string                     `json:"kind"`
+	ClusterName      string                     `json:"clusterName"`
+	ClusterShortName string                     `json:"clusterShortName"`
+	CustomerGUID     string                     `json:"customerGUID"`
+	ImagesCount      int                        `json:"imagesCount"`
+	SeverityStats    map[string]SeverityDetails `json:"severityStats"`
+	RiskFactors      []RiskFactor               `json:"riskFactors"`
+	Labels           []string                   `json:"labels"`
+}
+
+type SeverityDetails struct {
+	Total   int             `json:"total"`
```

Comment:
I think it is better to put the counter as fields 
like CriticalCount etc 
how do you send a sort by SeverityStats[Critical].total 
it is much simpler to sort by criticalCount:desc,hightCount:desc ...

if we do that than maybe we can make all this SeverityStats  simpler like map[string][]string (and name it CEVs)